src/java/boa/runtime/BoaRunner.java

```diff
@@ -82,7 +83,9 @@ public Job job(final Path[] ins, final Path out, final boolean robust) throws IO
 		// map output compression
 		configuration.setBoolean("mapred.compress.map.output", true);
 		configuration.set("mapred.map.output.compression.type", "BLOCK");
-		configuration.setClass("mapred.map.output.compression.codec", SnappyCodec.class, CompressionCodec.class);
```

Comment:
This might be stupid on my part but I don't see any method in Configuration class to reset or unset some class values. Could you please explain as for how can I reset this Configuration in the case of localMode execution directly from HadoopTemplate. Can I simply set the setClass to null?